Lean4Axiomatic/Natural/Impl/Generic/Exponentiation.lean

```diff
@@ -31,7 +37,7 @@ Anything to the zero power is one.
 recursive definition of `_pow`.
 -/
 theorem pow_zero {x : α} : x ^ (0:ℕ) ≃ 1 := calc
-  _ ≃ x ^ (0:ℕ)          := Rel.refl
+  _ ≃ x ^ (0:ℕ)              := Rel.refl
```

Comment:
Nitpick: fix alignment of the `:=` operators (there's a few more instances below too).